os/board/rtl8730e/src/rtl8730e_boot.c

```diff
@@ -131,6 +131,18 @@ void board_i2s_initialize(void)
 void board_spi_initialize(void)
 {
 #ifdef CONFIG_SPI
+
+#if defined(AMEBASMART_SPI0) && (AMEBASMART_SPI0 == 0)
```

Comment:
1. Maybe it's `CONFIG_.....`.
2. If it's `CONFIG_...`, no need `&& (.... == 0)`.